<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maksuperusteinen</class>
 <widget class="QDialog" name="Maksuperusteinen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maksuperusteinen arvonlisävero</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pic/pic.qrc">:/pic/varoitus.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ennen maksuperusteisen arvonlisäveron käyttöön ottamista lue huolellisesti maksuperusteisen arvonlisäveron ohjeet.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="alkaaCheck">
       <property name="text">
        <string>Maksuperusteinen arvonlisäverotus alkaa</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="alkaaDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="paattyyCheck">
       <property name="text">
        <string>Suoriteperusteiseen arvonlisäveroon palataan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="paattyyDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="ohjeNappi">
       <property name="text">
        <string>Ohje</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/ohje.png</normaloff>:/pic/ohje.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Maksuperusteinen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Maksuperusteinen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alkaaCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>alkaaDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paattyyCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>paattyyDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
